core/domain/blog_domain.py

```diff
@@ -725,9 +727,13 @@ def require_valid_displayed_author_name(
         """
         if not author_name:
             raise utils.ValidationError('Empty author name supplied.')
+        if len(author_name) < constants.MIN_AUTHOR_NAME_LENGTH:
+            raise utils.ValidationError(
+                'Author name should have at least %s characters.'
+                % constants.MIN_AUTHOR_NAME_LENGTH)
```

Comment:
Do we need to verify/ensure that existing blog posts on the server satisfy this criteria?

Reply:
Although the save button is not grayed out when attempting to save `authorName` with only one character, it cannot be saved in the backend due to the existing regex validation. Therefore, it is unlikely that any users on the server have an author name that is less than two characters, making verification unnecessary.

Reply:
Any updates on the review @kevintab95

Reply:
Could you please point me to the existing regex validation please?

Reply:
Sure, This is the regex which expects a minimum of 2 characters

https://github.com/oppia/oppia/blob/4def547d5425ff55f772ec2907885a7961ee330a/assets/constants.ts#L6243

And, this is the validation in place. PTAL

https://github.com/oppia/oppia/blob/4def547d5425ff55f772ec2907885a7961ee330a/core/domain/blog_domain.py#L732-L734